ui/static/ui/js/listing.js

```diff
@@ -80,6 +80,23 @@ define('listing',
 
       $('[data-toggle=popover]').popover();
 
+       $(document).keyup(function(e) {
```

Comment:
Please add a comment before this block saying 'Close panels on escape keypress'. You will also need to fix indentation here